controllers/articles.controller.js

```diff
@@ -0,0 +1,13 @@
+const { getArticlesById } = require("../models/articles.model");
+
+exports.getArticleById = (req, res, next) => {
```

Comment:
this info needs to be on a different branch as its task 4 and makes it hard to read when sorting through code 😢